shared/storage.ts

```diff
@@ -183,4 +183,10 @@ const getVCKeyName = (key: string) => {
   return key.split(':').splice(0, 4).join(':');
 };
 
+// To print the MMKV data cal this function in getItem
+// const getMMKVData = async () =>{
+//   const mmkvData = await MMKV.indexer.getKeys()
+//   console.log("mmkv data", mmkvData)
+// }
+
```

Comment:
why this commented code?

Reply:
This function is to get the data from MMKV. If anybody wants to check the MMKV data they can uncomment this and use it.

Reply:
Then, let's add it as uncommented one. No one is going to see commented code in future to uncomment and use

Reply:
uncommented it